### ISSUE TITLE
Update tile drag & drop styles

### ISSUE THREAD

**Issue by Blackbaud-AlexKingman** [blackbaud/skyux-tiles#27]
@Blackbaud-ToddRoberts commented on [Fri Jan 04 2019](https://github.com/blackbaud/skyux2/issues/2242)

Per approved drag & drop standarization proposal.

- Update drag handle icon (dependent on #1506).
- Update drop target styling.

https://blackbaud.invisionapp.com/share/7XO2FUJ5RHG#/screens/320047110

**Comment by Blackbaud-DanMiller** [comment]
Closing issue due to inactivity. If you believe this should still be addressed, please file a new issue and include any additional context and justification. Thank you.
